### PR TITLE
Enha: CI Update sentry-cli

### PR DESCRIPTION
**Pull request by denrase**
_#skip-changelog_

- [x] ~~Blocked: Need to merge #143 first.~~
- [x] Wait until https://github.com/getsentry/github-workflows/pull/18 is merged.

### CODE REVIEW

File: .github/workflows/update-deps.yml
Comment (philipphofmann) [inline .github/workflows/update-deps.yml:5]: `l`: Please add a comment when this runs. I think it's every day at 3 am UTC?

**Review by philipphofmann — COMMENTED** [review]
Nice, what's missing to finish this PR?

File: .github/workflows/update-deps.yml
Comment (vaind) [inline .github/workflows/update-deps.yml:10]:
```suggestion
      - master
```

**Comment by denrase** [comment]
@philipphofmann Alright, we now also automatically update and commit the binaries when an PR for a new `sentry-cli` version is created. We are using a fork of the workflow and need to wait until it is merged to finish this PR here.

File: .github/workflows/update-deps.yml
Comment (vaind) [inline .github/workflows/update-deps.yml:21]: btw ubuntu runners are more readily available and faster to start up, most of the times.

File: .github/workflows/update-deps.yml
Comment (vaind) [inline .github/workflows/update-deps.yml:14]: v1 is updated
```suggestion
    uses: denrase/github-workflows/.github/workflows/updater.yml@v1
```

**Review by vaind — CHANGES_REQUESTED** [review]

File: .github/workflows/update-deps.yml
Comment (vaind) [inline .github/workflows/update-deps.yml:25]: just wondering - can this condition be moved to the job? I've never used it that way but seems to be possible: https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution

Comment (denrase) [reply .github/workflows/update-deps.yml:25]: Yeah sounds good, less duplication. 👍

**Review by brustolin — APPROVED** [review]
LGTM

**Pull request merged**
